Integrations/JsonWhoIs/JsonWhoIs.py

```diff
@@ -100,38 +115,34 @@ def whois(url: str) -> tuple:
                        url_suffix='/api/v1/whois',
                        params=params)
 
-    # entry context by code convention
-    ec_temp = {
-        'DomainStatus': raw.get('status'),
-        'NameServers': list_by_ec(raw.get('nameservers'), key_needed=['name']),
-        'CreationDate': raw.get('created_on'),
-        'UpdatedDate': raw.get('updated_on'),
-        'ExpirationDate': raw.get('expires_on'),
-        'Registrar': dict_by_ec(raw.get('registrar')),
-        'Registrant': list_by_ec(raw.get('registrant_contacts'), key_needed=['name', 'phone', 'email']),
-        'Admin': list_by_ec(raw.get('admin_contacts'), key_needed=['name', 'phone', 'email'])
-    }
-
-    # Clear unused fields
-    ec_temp = createContext(ec_temp, removeNull=True)
-
     # Build all ec
     ec = {
         'Domain': {
-            'WHOIS': ec_temp
+            'WHOIS': {
```

Comment:
not what i meant... lets discuss on Sunday.

Reply:
Changed as we discussed